tests/ArmTemplates.Tests/Extractor/Scenarios/SchemaExtractorTests.cs

```diff
@@ -0,0 +1,69 @@
+﻿// --------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+// --------------------------------------------------------------------------
+
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Microsoft.Azure.Management.ApiManagement.ArmTemplates.Commands.Executors;
+using Microsoft.Azure.Management.ApiManagement.ArmTemplates.Common.Constants;
+using Microsoft.Azure.Management.ApiManagement.ArmTemplates.Common.Templates.Builders;
+using Microsoft.Azure.Management.ApiManagement.ArmTemplates.Extractor.EntityExtractors;
+using Microsoft.Azure.Management.ApiManagement.ArmTemplates.Extractor.Models;
+using Microsoft.Azure.Management.ApiManagement.ArmTemplates.Tests.Extractor.Abstractions;
+using Microsoft.Azure.Management.ApiManagement.ArmTemplates.Tests.Moqs.ApiClients;
+using Xunit;
+
+namespace Microsoft.Azure.Management.ApiManagement.ArmTemplates.Tests.Extractor.Scenarios
+{
+    [Trait("Category", "Schemas Extraction")]
+    public class SchemaExtractorTests: ExtractorMockerWithOutputTestsBase
+    {        
+        public SchemaExtractorTests() : base("schemas-tests")
+        {
+        }
+
+        [Fact]
+        public async Task GenerateSchemasTemplates_ProperlyParsesResponse()
+        {
+            // arrange
+            var currentTestDirectory = Path.Combine(this.OutputDirectory, nameof(GenerateSchemasTemplates_ProperlyParsesResponse));
+
+            var extractorConfig = this.GetDefaultExtractorConsoleAppConfiguration();
+            var extractorParameters = new ExtractorParameters(extractorConfig);
+            var fileLocation = Path.Combine("ApiClientJsonResponses", "ApiManagementListSchemas_success_response.json");
```

Comment:
can we also move `ApiClientJsonResponses` constant to base class? So that we will not ever mess up with writing not proper string. 

We can still use it with Path.Combine here

Reply:
thank you for noticing! 
Updated